web/src/views/DandisetLandingView/DandisetLandingView.vue

```diff
@@ -69,7 +70,7 @@
             <DandisetMain
               :schema="schema"
               :meta="meta"
-              @edit="edit = true"
+              @edit="edit = true; readonly = $event"
```

Comment:
I don't like this, since it seems the only reason this is here is to initialize `readonly` to the correct value the first time the view/edit button is clicked. I think we should instead move the `userCanModifyDandiset` async computed prop up to this level (probably renaming it to something like `readonly`), and then pass it down equally to the `DandisetMain` and `Meditor` components.

Reply:
See 1b88930f68b314e620fd17e90306f4512cf5bbc3.